src/main/java/com/crudtest/model/User.java

```diff
@@ -13,10 +20,17 @@
 
         @Column(name = "FIRST_NAME")
         private String firstName;
+
         @Column(name = "LAST_NAME")
         private String lastName;
 
-        public User(){}
+
+        @Column(name = "DATE_CREATED", updatable = false)
+        private Date dateCreated = new Date();
```

Comment:
Please move the initialization of date created either to the constructor or the the user service